ComputeClient/Compute.Contracts/Network20/ServerBaseType.cs

```diff
@@ -57,8 +57,25 @@ public partial class ServersSummaryResposeType
     [System.Xml.Serialization.XmlRootAttribute("server", Namespace = "urn:didata.com:api:cloud:types", IsNullable = false)]
     public partial class ServerDetailType : ServerBaseType
     {
+        /// <remarks/>
+        public NameValuePairType[] advancedVirtualizationSettingField;
```

Comment:
Would prefer if these fields were private for better encapsulation but this pattern of having public fields seems common in the existing codebase so didn't want to change that.